src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNegativeInfinitySymbol.cs

```diff
@@ -12,8 +12,8 @@ public class NumberFormatInfoNegativeInfinitySymbol
         public static IEnumerable<object[]> NegativeInfinitySymbol_TestData()
         {
             yield return new object[] { NumberFormatInfo.InvariantInfo, "-Infinity" };
-            yield return new object[] { new CultureInfo("en-US").NumberFormat, NumberFormatInfoData.GetNegativeInfinitySymbol("en-US") };
-            yield return new object[] { new CultureInfo("fr-FR").NumberFormat, NumberFormatInfoData.GetNegativeInfinitySymbol("fr-FR") };
+            yield return new object[] { new CultureInfo("en-US").NumberFormat, "-" + NumberFormatInfoData.GetPositiveInfinitySymbol("en-US") };
```

Comment:
ditto. format a float negative infinity number using the desired culture and use the result as the symbol you are checking against.